README.md

```diff
@@ -30,14 +34,31 @@ Or install it yourself as:
 
 ## Usage
 
+### Update sidekiq.yml
+A queue for each instance should be created, add the following to your processing queues in `sidekiq.yml`, you need to
+configure the `queue_name` on SidekiqAlive to match.
+
+```yaml
```

Comment:
In kubertenes the replicas can be autoscaled I do not see a way to place a config file in this setup.
ideas?

Reply:
you can create a configmap that gets loaded to every pod that creates the sidekiq.yml file for you. then mount the ConfigMap as the sidekiq.yml eg: 

ConfigMap
```
apiVersion: v1
kind: ConfigMap
metadata:
  name: sidekiq-default-conf
data:
  config: |-
    ---
    :pidfile: tmp/pids/sidekiq.pid
    :logfile: log/sidekiq.log
    :max_retries: 0
    :concurrency: 4
    :queues:
      - [default, 1]
      - [sidekiq_alive, 1]
    :dynamic: true
    :timeout: 300
```
then mount the file in the pod/deployment

```
      volumes:
        - name: sidekiq-default
          configMap:
            name: sidekiq-default-conf
            items:
              - key: config
                path: sidekiq.yml

```

Reply:
Great gem, by the way.. ! @arturictus